Makefile.envs

```diff
@@ -271,6 +272,7 @@ ifeq ($(DISABLE_DYLINK), 1)
 		,HEAPF64
 	export MAIN_MODULE_LDFLAGS= $(LDFLAGS_BASE) \
 		-s MODULARIZE=1 \
+		-s RELOCATABLE=1 \
```

Comment:
What's this for? Isn't it implied by `-s MAIN_MODULE=1`?

Reply:
It was needed for `addFunction` (unless RELOCATABLE=1 is set, the `wasmTable` is null), but we don't use it anymore, so I think I can remove it. Thanks for the catch!